src/style.css

```diff
@@ -0,0 +1,27 @@
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+h1,
+h2,
+p,
+ol {
+  margin: 0;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
I am reset default browser styles for all tags